packages/sku/config/typescript/tsconfig.js

```diff
@@ -44,16 +33,23 @@ module.exports = () => {
       lib: ['dom', 'es2022'],
       target: 'es2022',
     },
-    include: includePaths,
   };
 
+  if (!paths?.appSkuConfigPath?.endsWith('.ts')) {
+    // The placeholder file needs to be included so we don't break JS only projects.
+    // See the comments in placeholder.ts
+    config.include = [
+      path.join(__dirname, '../../lib/placeholder.ts'),
+      ...(config.include || []),
+    ];
+  }
```

Comment:
Did we decide to remove this, in favour of just converting `sku.config.js` to `sku.config.ts`?

Reply:
Yes, I forgot to remove it.